sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -505,8 +506,7 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       }
       // Find the correct user type and role.
       const authType = await this.institutionUserTypeAndRoleRepo.findOne({
-        type: permission.userType,
-        role: permission.userRole ?? null,
+        where: { type: permission.userType, role: permission.userRole ?? null },
```

Comment:
Same comment from before. Should we use `IsNull()` instead?

Reply:
It did not throw any error or warnings, but thought of taking these changes in the next PR. But will change it as its comments are given now.